Patch/Kdlsym Tweaks

Al-Azif opened the pull request:
- Kdlsym/Patches Tweaks
  - Added missing patches from previous firmwares (6.72)
  - Removed printf kernel patches for 4.05-5.05 whose need is preempted by another patch
  - Standardized Kdlsym/Patches to match the others (4.05-6.72)

Note: This assumes DeathRGH's second ptrace patch is correct

Al-Azif committed:
Fix old ptrace patches

Al-Azif committed:
Remove patches that are prempted by another

Al-Azif committed:
Added missing patches from previous firmwares

Al-Azif committed:
kdlsym tweaks

Al-Azif committed:
Last tweaks

Al-Azif committed:
Missed one

Al-Azif committed:
Add ssc_enable_data_mount_patch

Al-Azif committed:
Fix two offsets

kiwidoggie reviewed (changes requested):
See requested changes, feel free to ping/fill me in on discord too on the things that I don't know why the patches were changed

Al-Azif commented:
I believe I addressed all the issues, The comments weren't so much removed but rather I made the patch files from scratch and just didn't re-add them as it made diffing the file a pain in the ass. I'll add the comments to every FW when I get time.

I was going for consistency on this pull as it's harder to get a hold of what's going on and compare stuff when the FW's Patches/Kdlsym files are jumbled messes from one another and some include some.
